components/registry-facade/pkg/registry/imagecfg.go

```diff
@@ -27,6 +27,14 @@ type ImageSpecProvider interface {
 	GetSpec(ctx context.Context, ref string) (*api.ImageSpec, error)
 }
 
+// FixedImageSpecProvider provides a single spec
+type FixedImageSpecProvider api.ImageSpec
+
+func (p FixedImageSpecProvider) GetSpec(ctx context.Context, ref string) (*api.ImageSpec, error) {
```

Comment:
Should we make p a pointer? `*p`?

Reply:
That would make the use of the `FixedImageSpecProvider` more cumbersome. It's unfortunate that the message contains a mutex, but copying it is inconsequential here.